packages/terra-form-select/src/Select.scss

```diff
@@ -0,0 +1,118 @@
+:local {
+  .select {
+    background-color: #fff;
+    background-image: var(--terra-select-focus-animation-background, none);
+    background-position: 0 100%;
+    background-repeat: no-repeat;
+    background-size: var(--terra-select-focus-background-start, 0 0);
+    border-color: var(--terra-select-input-border-color, #dedfe0);
+    border-radius: 3px;
+    border-style: solid;
+    border-width: 1px;
+    cursor: pointer;
+    font-size: var(--terra-select-font-size, 1em);
+    line-height: var(--terra-select-line-height, 1.07);
+    padding: var(--terra-select-input-padding, 0.571em 0.5em);
+    transition-duration: var(--terra-select-focus-transition-duration, 0s);
+    transition-property: background-size;
+    transition-timing-function: ease;
+    width: 100%;
+
+    &.is-disabled {
+      background-color: var(--terra-select-disabled-background-color, #f4f4f4);
+      border-color: var(--terra-select-disabled-border-color, #dedfe0);
+      cursor: default;
+
+      &:focus {
+        background-size: 0 0;
+        border-color: var(--terra-select-disabled-border-color, #dedfe0);
+      }
+
+      &:hover {
+        border-color: var(--terra-select-disabled-border-color, #dedfe0);
+      }
+    }
+
+    &:hover {
+      border-color: var(--terra-select-input-hover-border-color, #4cb2e9);
+    }
+
+    &:focus,
+    &.open {
+      background-size: var(--terra-select-focus-animation-size, 0 0);
+      border-color: var(--terra-select-input-focus-border-color, #4cb2e9);
+      box-shadow: var(--terra-select-focus-box-shadow, 0 0 7px 4px rgba(76, 178, 233, 0.35), 0 0 1px 3px rgba(76, 178, 233, 0.5), 0 0 1px 0 #fff);
```

Comment:
Do we anticipate box-shadows that appear at places other than 0 0? If so, we should encapsulate this with directional attribute values like we have here. https://github.com/cerner/terra-core/blob/a6033d7e3670cea80497408167673168fd5d38f9/packages/terra-form-textarea/src/Textarea.scss#L57